progression/src/main/java/me/mykindos/betterpvp/progression/progression/perks/DropMultiplier.java

```diff
@@ -0,0 +1,20 @@
+package me.mykindos.betterpvp.progression.progression.perks;
+
+import me.mykindos.betterpvp.core.config.Config;
+import me.mykindos.betterpvp.core.utilities.UtilMath;
+import net.kyori.adventure.text.Component;
+import org.bukkit.Bukkit;
+
+import java.util.Random;
+
+public interface DropMultiplier {
+
+    default int getExtraDrops(double chance) {
+        int multiplier = (int) Math.floor(chance/100);
+        double newChance = chance % 100;
+        if (UtilMath.randomInt(0, 100) < newChance) {
+            multiplier++;
+        }
+        return multiplier;
+    }
```

Comment:
In general this whole function would double the drops 100% of the time for higher levels. I'm not sure if we want to do that since some fish can be quite heavy and having it multiply will give effectively give them double the money

Reply:
Intended functionality, if this behaviour is not wanted it is possible to achieve that via config.